age.md

```diff
@@ -273,6 +275,79 @@ other types. This is to uphold an expectation of authentication that is
 implicit in password-based encryption. The identity implementation MUST reject
 headers where an scrypt stanza is present alongside any other stanza.
 
+### The p256tag recipient type
+
+Age does not generate p256tag identities, but supports encrypting to
+p256tag recipients.
+
+The p256tag recipient is computed as 
+
+    recipient = P256(identity, basepoint)
+
+P256 is the scalar multiplication over P-256, with SEC 1 point decoding and 
+encoding.
+
+The recipient is encoded as Bech32 with HRP `age1p256tag`.
+
+    age1p256tag1qf0l9gks6x65ha077wq3w3u8fy02tpg3cd9w5j0jlgpfgqkcut2lwpmkcwq
```

Comment:
The plugin protocol requires that this recipient be handled by the `age-plugin-p256tag` plugin and not by the native app, which I don't think is the intention. If we want to define a new recipient encoding that all age implementations are required to understand, it likely needs to be separate from the plugin namespace.